api/tests/conftest.py

```diff
@@ -459,6 +459,17 @@ def get(self, route: str, params=None, headers=None):
         self._print_spec("GET", route, None, result)
         return result
 
+    def faulty_get(self, route: str, headers=None, raw_json=None):
```

Comment:
```suggestion
    def get_with_invalid_json_body(self, route: str, headers=None, raw_json=None):
```
(Simple proposition, c'est ouvert).
Je ne suis généralement pas fan des noms long mais je pense que `faulty_get` est un peu large.

Reply:
Tu as raison c'est changé.